roles/sriov_config/tasks/create_node_policies.yml

```diff
@@ -1,11 +1,17 @@
 ---
+# OCPBUGS-32139 workaround suggests to retry the creation of the SRIOV policy if failing.
+# Trying this during 1 minute in case any of them fails.
 - name: Create SriovNetworkNodePolicy
   community.kubernetes.k8s:
     definition: "{{ lookup('template', 'templates/sriov-network-node-policy.yml.j2') }}"
   loop: "{{ sriov_network_configs }}"
   loop_control:
     loop_var: sriov
     label: "{{ sriov.resource }}"
+  retries: 6
+  delay: 1
```

Comment:
Hi @ramperher I believe like this it's a 'one second' delay, not 'one minute'. To have 1 minute, you can go with
```
 delay: 60
```

Reply:
Oh, sorry, I deleted a 0 in my last refactor of the change, I wanted 6 retries with 10 seconds delay.